File: src/php/playground/retrieve-rate.php
<?php
require_once '/Users/jhammond/git/easypost/easypost-tools/vendor/autoload.php';
use EasyPost\EasyPost;
use EasyPost\Error;
use EasyPost\Rate;
use Dotenv\Dotenv;

$dotenv = Dotenv::createImmutable('/Users/jhammond/git/easypost/easypost-tools');
$dotenv->load();
EasyPost::setApiKey(getenv('EASYPOST_PROD_API_KEY'));

try {
    $rate = Rate::retrieve('rate_675afb1d32054566b6bc2760f9a76902');
    echo $rate;
} catch (Error $exception) {
    echo $exception;
}

File: src/python/shipments/retrieve_rate.py
import os

import easypost
from dotenv import load_dotenv

load_dotenv()
easypost.api_key = os.getenv('EASYPOST_PROD_API_KEY')

rate = easypost.Rate.retrieve('rate_675afb1d32054566b6bc2760f9a76902')

print(rate)
